app/policies/activity_policy.rb

```diff
@@ -47,8 +47,10 @@ def resolve
     end
   end
 
-  private def editable_report_for_organisation_and_fund
+  private
+
+  def editable_report?
     fund = record.associated_fund
-    Report.editable.find_by(organisation: record.organisation, fund: fund)
+    Report.editable.exists?(organisation: record.organisation, fund: fund)
```

Comment:
Or, to replace my previous comment, `Report.editable.for_activity(record).exists?`

Reply:
I couldn't do this in the end, as I needed to do: 

`Report.editable.where(..., organisation: [record.extending_organisation, record.organisation]).exists?`